protos/apigw/v1/apigw.proto

```diff
@@ -82,6 +82,12 @@ message TerraformEntity {
   TerraformEntityMethodType type = 1;
   string name = 2;
   bool json = 3;
+  int32 operation_number =  4;
+  
+  oneof optionalExclusion {
```

Comment:
is this more a enum of onlys?  is there ever a chance for 3 things, where you might want 2 of 3?

Reply:
I guess maybe enum makes sense since the options are just bools? But no, I don't think it makes sense that there would ever be more than just the two options here. You either want just a datasource, just a resource, or both (which is just the absence of this being set.